modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,258 @@
+= ownCloud on QNAP
+:toc: right
+:toclevels: 1
+
+== Introduction
+
+If you have a QNAP NAS, you will soon be able to install the ownCloud app from the App Center and enjoy a ready-to-use ownCloud. The free-of-charge community edition can be used with up to five users and an unlimited number of guest users.
+
+In case you want to use Enterprise features, you'll need to purchase a license from the QNAP store, where you can also obtain licenses for additional users, after you've bought a license for the initial five.
```

Comment:
```suggestion
If you want to use Enterprise features, you'll need to purchase a license from the QNAP store, where you can also obtain licenses for additional users, after you've bought a license for the initial five.
```

"if" sounds more natural in English than "in case"